Add metaomiocs nf-test fest event (#3370)
nf-core/website: sites/main-site/src/content/events/2025/SIG_metaomics_nftestfest.md

---
title: meta-omics nf-test fest

> subtitle: Hackathon for implementing nf-test in meta-omics related pipelines!

Reviewer: ```suggestion
subtitle: Hackathon to implement nf-test in meta-omics related pipelines!
```
Actually in the style docs: "Avoid gerund" (https://nf-co.re/docs/guidelines/documentation/writing_style).

---
title: meta-omics nf-test fest
subtitle: Hackathon for implementing nf-test in meta-omics related pipelines!
type: training
startDate: "2025-06-11"

> startTime: "09:00 CEST"

Reviewer 2: ```suggestion
startTime: "09:00+02:00"
```

endDate: "2025-06-12"
endTime: "17:00 CEST"
locations:
  - name: gather.town

>     links: "#organisational-details"

Reviewer 2: [![suggestion](https://img.shields.io/badge/suggestion-3B82F6)](https://pullpo.io/cc?l=suggestion) 
double check this

>     links: "#organisational-details"

Author: ```suggestion
```
